Dockerfile

```diff
@@ -25,7 +28,7 @@ RUN apt-get update \
         libboost-system${boost_dev_version}-dev \
         libeigen3-dev \
         libfreetype6-dev \
-        postgresql-server-dev-${postgres_version}=$(postgres -V | awk '{print $3}')\* \
+        postgresql-server-dev-all \
```

Comment:
I didn't know it could be simplified like this, thank you so much.

Reply:
no this will install a lot of versions of postgres. I am building against v12 and this will install 11-16. Then cmake will pick version 16 in part coz it's in the PATH and in part coz of the dockerfile overriding postgres arg in the latter build step.

Reply:
fixed it in https://github.com/rvianello/docker-postgres-rdkit/pull/4